### PR TITLE
Inputs

### PR DESCRIPTION
**Pull request by vanshitalavange**
Created input component of the following types:
- labeled input
- input with helper message
- numbers only input
- text area inputs
- radio inputs
- checkbox inputs
- file input
- select options input
- date input
- input with icon
- read-only input
https://user-images.githubusercontent.com/86543154/152421175-2df1f451-1fd3-4456-b16f-88f0f8796546.mp4

### CODE REVIEW

File: stylesheets/components/input.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.form-group {
+  background-color: var(--gunmental-color);
+  border: 1px solid var(--gravel-color);
+  padding: 1.5rem 1.5rem;
```
Comment (ttahm3d) [inline stylesheets/components/input.css]: Use shorthand here. 

[Read More](https://www.w3schools.com/css/css_padding.asp)

**Review by ttahm3d — COMMENTED** [review]
Looks good. Just use shorthand for padding if the padding is same on all the sides

**Review by vivekmehto — COMMENTED** [review]
Looks good. just a suggestion you can use cursor not allowed on disabled inputs.

File: stylesheets/components/input.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/components/input.css]: Changes done, thank you

**Pull request merged**